### PR TITLE
Simplify Astro README

### PR DESCRIPTION
**Pull request by tatiana**
Solves #101, #149, #158, #159

In summary:
* Refactor README (#101)
* Add contribution guidelines
* Add code of conduct
* Add security guidelines
* Add changelog
* Support `load_file` from HTTP endpoint
* Add SQLite example (#149)
* Fix SQLite `load_file` (#158)
* Fix SQLite `transform` (#159)

### CODE REVIEW

File: example_dags/example_sqlite_load_transform.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+from datetime import datetime
```
Comment (utkarsharma2) [inline example_dags/example_sqlite_load_transform.py:1]: Should we add this dag to CI?

File: src/astro/sql/operators/sql_decorator.py [hunk 2]
```diff
@@ -6,30 +6,27 @@
 You may obtain a copy of the License at
 
     http://www.apache.org/licenses/LICENSE-2.0
-
+from sqlalchemy import create_engine
```
Comment (utkarsharma2) [inline src/astro/sql/operators/sql_decorator.py:9]: is this a typo?

File: example_dags/example_sqlite_load_transform.py (hunk 1 above)
Comment (tatiana) [reply example_dags/example_sqlite_load_transform.py:1]: great catch, @utkarsharma2 , I add it!

File: src/astro/sql/operators/sql_decorator.py (hunk 2 above)
Comment (tatiana) [reply src/astro/sql/operators/sql_decorator.py]: :joy: completely, a traditional `CTRL-C` + `CTRL-V` mistake

File: src/astro/sql/operators/sql_decorator.py [hunk 3]
```diff
@@ -237,74 +238,36 @@ def _set_schema_if_needed(self):
...
-
-    def get_sqlite_hook(self):
-        return SqliteHook(
-            sqlite_conn_id=self.conn_id,
-        )
-
-    def get_postgres_hook(self):
-        return PostgresHook(postgres_conn_id=self.conn_id, schema=self.database)
-
     def _run_sql_string(self, sql, parameters):
```
Comment (utkarsharma2) [inline src/astro/sql/operators/sql_decorator.py:241]: I think we don't need  _run_sql_string() any more, can we replace it with -  _run_sql_alchemy_obj.

Comment (tatiana) [reply src/astro/sql/operators/sql_decorator.py]: Done!

**Review by utkarsharma2 — APPROVED** [review]

**Comment by tatiana** [comment]
I noticed that once I added the SQLite tests to tests_examples, one of our test workers started hanging.
I disabled and I'm logging a separate ticket to investigate this

**Pull request merged**